BUILD.bazel

```
@@ -0,0 +1,325 @@
+load(
+    "//ign_bazel:cmake_configure_file.bzl",
+    "cmake_configure_file",
+)
+load(
+    "//ign_bazel:generate_include_header.bzl",
+    "generate_include_header",
+)
+load(
+    "//ign_bazel:ign_export_header.bzl",
+    "ign_export_header",
+)
+load(
+    "//ign_bazel:generate_yaml.bzl",
+    "generate_yaml",
+)
+
+load(
+    "@bazelruby_rules_ruby//ruby:defs.bzl",
+    "ruby_binary",
+    "ruby_library",
+    "ruby_test",
+    "ruby_rspec",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+PROJECT_NAME = "ignition-launch"
+PROJECT_MAJOR = 3
+PROJECT_MINOR = 0
+PROJECT_PATCH = 0
+IGN_DESIGNATION = "/tmp"
+CMAKE_INSTALL_PREFIX = "/tmp"
+IGN_DATA_INSTALL_DIR = "/tmp"
+IGN_LAUNCH_PLUGIN_INSTALL_PATH = "./ign_launch"
+library_location = "/usr"
+IGN_LIBRARY_PATH = "/usr"
```

In the other projects, we have been doing something like this:

```suggestion
CMAKE_INSTALL_PREFIX = "./ign_launch"
IGN_DATA_INSTALL_DIR = "./ign_launch"
IGN_LAUNCH_PLUGIN_INSTALL_PATH = "./ign_launch"
library_location = "./ign_launch"
IGN_LIBRARY_PATH = "./ign_launch"
```